### PR TITLE
Abi kvm compact

### PR DESCRIPTION
**Pull request by isumitsolanki**
This PR will resolve below mentioned issues. 
ABI Compact Support Implementation --  https://github.com/IBM/Ansible-OpenShift-Provisioning/issues/322
Duplicate entries found in haproxy.cfg  -- https://github.com/IBM/Ansible-OpenShift-Provisioning/issues/321
SNO install is broken  -- https://github.com/IBM/Ansible-OpenShift-Provisioning/issues/315

### CODE REVIEW

File: playbooks/5_setup_bastion.yaml [hunk 1]
```diff
@@ -89,7 +89,6 @@
     - { role: dns, when: env.bastion.options.dns }
     - { role: check_dns, when: env.bastion.options.dns is defined and env.bastion.options.dns }
     - { role: haproxy, when: env.bastion.options.loadbalancer.on_bastion }
-    - { role: sno_haproxy, when: env.bastion.options.loadbalancer.on_bastion and control_node_count | int == 1 }
```
[the remark this answers is not in the document]
Comment (isumitsolanki) [reply playbooks/5_setup_bastion.yaml:92]: Sure @AmadeusPodvratnik

[the remark this answers is not in the document]
Comment (isumitsolanki) [reply playbooks/5_setup_bastion.yaml:92]: I just kept it for future reference if needed.

**Review by AmadeusPodvratnik — APPROVED** [review]
Hi @isumitsolanki Thank you. 
/lgtm

**Pull request merged**